project/images/official/editable/429.svg

```diff
@@ -110,7 +110,7 @@
        dominant-baseline="middle"
        text-anchor="middle"
        font-family="sans-serif"
-       font-size="40px"
```

Comment:
should this be even bigger to match 4.27?

Reply:
![image](https://user-images.githubusercontent.com/50236352/146943779-0abcdbf5-a27f-4d07-a1be-68d26a500b92.png)

I did to get in "La Chaud-de-Fonds"

Reply:
But the size should be the same in both no?

Reply:
![image](https://user-images.githubusercontent.com/50236352/146954755-d4602fac-4335-4335-bdf2-335b15918614.png)

It's almost the same